build_library/qemu_template.sh

```diff
@@ -166,8 +155,12 @@ if [ "${SAFE_ARGS}" -eq 1 ]; then
 else
     case "${VM_BOARD}+$(uname -m)" in
         amd64-usr+x86_64)
+            set -- -global ICH9-LPC.disable_s3=1 \
+                   -global driver=cfi.pflash01,property=secure,value=on \
+                   "$@"
             # Emulate the host CPU closely in both features and cores.
-            set -- -machine q35,accel=kvm:hvf:tcg,$smmarg -cpu host -smp "${VM_NCPUS}" "$@" ;;
+            set -- -machine q35,accel=kvm:hvf:tcg,smm=on -cpu host -smp "${VM_NCPUS}" "$@"
```

Comment:
@jepio I've updated to drop the SECURE_BOOT vars, as because if the variable is empty, the firmware would still be the one with SECURE_BOOT enabled, and also has SMM support that means that we would need to disable S3. So, I dropped the SECURE_BOOT. What do you think? This is to address @krnowak comments (https://github.com/flatcar/scripts/pull/1589/files/c891c7e550df4d1f9d6ffab87c28849c5e1b7d2a#r1469337359)

Reply:
Yes, tested and works fine. I'll squash this with the actual commit.